pkg/ldd/ldso_linux.go

```diff
@@ -9,15 +9,17 @@ import (
 	"path/filepath"
 )
 
-const ldso = "/lib*/ld-*.so.*"
-
-func LdSo() (string, error) {
-	n, err := filepath.Glob(ldso)
-	if err != nil {
-		return "", err
-	}
-	if len(n) == 0 {
-		return "", fmt.Errorf("No ld.so matches %v", ldso)
+// This is a real mess. Ubuntu keeps makimg it worse.
+func LdSo(bits int) (string, error) {
```

Comment:
Can you add at least one test for this function?

Reply:
phew. That gets hard b/c the systems we're on are all over the map. But I'll try.

Reply:
it's sort of implicitly tested with the ldd of /bin/date, and the variance in systems makes it tough. What will happen riscv for example? I have no idea. Will /lib, /lib32, and /lib64 be there? not sure I want to go beyond what's there.

Reply:
Yes, makes sense.